compliance_checker/suite.py

```diff
@@ -584,6 +584,10 @@ def standard_output(self, ds, limit, check_name, groups):
         print("\n")
         print("-" * width)
         print("{:^{width}}".format("IOOS Compliance Checker Report", width=width))
```

Comment:
I realize this was legacy code included with the project, but I'm not really a fan of these repeated print functions and mutliple calls to format for centering text in a plaintext table.   For things like f-strings, multiple levels could even be an injection hazard with user supplied strings.

  Could you combine the text to be centered into a single multiline string and then call `str.center(width)` instead?

Reply:
Should be fixed to satisfaction with latest commit